src/main/java/org/opensearch/ad/AnomalyDetectorExtension.java

```diff
@@ -91,6 +107,12 @@ private static ExtensionSettings initializeSettings() throws IOException {
         return settings;
     }
 
+    public OpenSearchClient getClient() throws IOException {
+        SDKClient sdkClient = new SDKClient();
+        OpenSearchClient client = sdkClient.initializeClient("localhost", 9200);
```

Comment:
Let's see if we can fetch these parameters from somewhere else.

Reply:
Raised https://github.com/opensearch-project/opensearch-sdk-java/pull/207 for the same